eap/eap64-amq-persistent-s2i.json

```diff
@@ -6,10 +6,10 @@
             "iconClass": "icon-eap",
             "tags": "eap,javaee,java,jboss",
             "version": "1.4.10",
-            "openshift.io/display-name": "JBoss EAP 6.4 + A-MQ (with https)",
+            "openshift.io/display-name": "JBoss EAP 6.4 + AMQ (Persistent)",
```

Comment:
should be actually "JBoss A-MQ" as an approved product abbreviation

Reply:
I think the hyphen was removed a couple of years ago.  Looking at the product page, it appears to be Red Hat AMQ now.  I suppose we should get some feedback from the PM.